### PR TITLE
added xenial.xml

### PR DESCRIPTION
**Pull request by z3dm4n**
Added config template for Ubuntu Xenial 16.04.

### CODE REVIEW

**Comment by z3dm4n** [comment]
The template is pretty much the same as stretch.xml, but for Xenial we are using libapache2-mod-fastcgi (like in jessie.xml) and I also fixed `a2dismod php7.0` to `a2dismod php`.

**Comment by Dessa** [comment]
is this tested?

i honestly wanted to work on 18.04 straight away, but if its already there we might as well just use it; i'll take a look.

**Comment by z3dm4n** [comment]
I would consider it work in progress, but I am going to test it thoroughly in the upcoming days.

**Comment by z3dm4n** [comment]
I now consider `xenial.xml ` soundly tested and ready for merge into master. What else do you need?

File: lib/configfiles/xenial.xml [hunk 1]
```diff
@@ -4631,7 +4631,7 @@ aliases:     files
 						<command><![CDATA[useradd -s /bin/false -g {{settings.system.mod_fcgid_httpgroup}} {{settings.system.mod_fcgid_httpuser}}]]></command>
 						<command><![CDATA[mkdir -p {{settings.system.mod_fcgid_configdir}}]]></command>
 						<command><![CDATA[mkdir -p {{settings.system.mod_fcgid_tmpdir}}]]></command>
-						<command><![CDATA[a2dismod php]]></command>
+						<command><![CDATA[a2dismod php7.0]]></command>
```
Comment (d00p) [inline lib/configfiles/xenial.xml]: A few lines below (php-fpm section), same line

**Pull request merged**